src/components/Popup/Popup.component.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './styles/popup.scss';
+
+const PopupComponent = ({
```

Comment:
Why did you create a popup component? 👀 

We do have material-ui and bootstrap. You literally have options there, and you chose to reinvent the wheel hahaha. Please revert this change, we don't need a popup.